icon_validator/rules/plugin_validators/runtime_validator.py

```diff
@@ -0,0 +1,72 @@
+import os
+import glob
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class RuntimeValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_setup(spec):
+        if "setup.py" in os.listdir(spec.directory):
+            with open(f"{spec.directory}/setup.py", "r") as file:
+                setup_str = file.read().replace("\n", "")
+
+                if f'install_requires=["insightconnect-plugin-runtime"]' not in setup_str\
+                        and f"install_requires=['insightconnect-plugin-runtime']" not in setup_str:
```

Comment:
```suggestion
                if 'install_requires=["insightconnect-plugin-runtime"]' not in setup_str\
                        and "install_requires=['insightconnect-plugin-runtime']" not in setup_str:
```

f'' is not required here.